docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -18,96 +18,106 @@ Nothing fancy, just plain CSS.
 
 {{"demo": "pages/guides/interoperability/StyledComponents.js", "hideToolbar": true}}
 
-[![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/plain-css-mtzri)
+[![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/plain-css-1k9wb)
 
-**PlainCssButton.css**
+**PlainCssSlider.css**
 
 ```css
-.button {
-  background-color: #6772e5;
-  color: #fff;
-  box-shadow: 0 4px 6px rgba(50, 50, 93, 0.11), 0 1px 3px rgba(0, 0, 0, 0.08);
-  padding: 7px 14px;
+.slider {
+  color: #20b2aa;
 }
-.button:hover {
-  background-color: #5469d4;
+.slider:hover {
+  color: #2e8b57;
 }
 ```
 
-**PlainCssButton.js**
+**PlainCssSlider.js**
 
 ```jsx
 import * as React from 'react';
-import Button from '@material-ui/core/Button';
-import './PlainCssButton.css';
+import Slider from '@material-ui/lab/SliderStyled';
+import './PlainCssSlider.css';
 
-export default function PlainCssButton() {
+export default function PlainCssSlider() {
   return (
     <div>
-      <Button>Default</Button>
-      <Button className="button">Customized</Button>
+      <Slider defaultValue={30} />
+      <Slider className="slider" defaultValue={30} />
     </div>
   );
 }
 ```
 
 ### Controlling priority ⚠️
 
-**Note:** JSS injects its styles at the bottom of the `<head>`. If you don't want to mark style attributes with **!important**, you need to change [the CSS injection order](/styles/advanced/#css-injection-order), as in the demo:
+**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the HTML `<head>`. If you don't want to mark style attributes with **!important**, you need to change the CSS injection order. Here's a demo of how it can be done for the default style engine - emotion:
 
 ```jsx
-import { StylesProvider } from '@material-ui/core/styles';
+import * as React from 'react';
+import { CacheProvider } from '@emotion/core';
+import createCache from '@emotion/cache';
+
+const head = document.getElementsByTagName('head')[0];
+
+const emotionContainer = head.insertBefore(
+  document.createElement('STYLE'),
+  head.firstChild,
+);
 
-<StylesProvider injectFirst>{/* Your component tree.
-      Now, you can override Material-UI's styles. */}</StylesProvider>;
+const cache = createCache({
+  container: emotionContainer,
+});
+
+export default function PlainCssPriority() {
+  return (
+    <CacheProvider value={cache}>
+      {/* Your component tree. Now you can override Material-UI's styles. */}
+    </CacheProvider>
+  );
+}
```

Comment:
@Andarist is there a better API for specifying that emotion needs to be the first element in the `head`?

Reply:
In the upcoming v11 (we have already released RC version) there is an additional `prepend` option that you could use.

Reply:
Awesome, I already have a branch testing the RC, will add a reminder there to update this guide when we upgrade 👍

Reply:
Thanks for taking a look btw :)